torchtitan/float8_linear.py

```diff
@@ -21,6 +21,9 @@
 from torchtitan.config_manager import JobConfig
 from torchtitan.logging_utils import logger
 
+# Float8 is only supported on H100+ GPUs
+SM90OrLater = torch.cuda.is_available() and torch.cuda.get_device_capability() >= (9, 0)
```

Comment:
borrowed from `torch/testing/_internal/common_cuda.py` https://fburl.com/xh717fpj

I want to avoid dependency on `torch/testing`, so copied it here